.gitignore

```diff
@@ -29,3 +29,4 @@ node_modules
 
 # local config file that contains passwords
 config/local.json
+config/*.local.json
```

Comment:
This allows us to keep multiple DB configs without committing them to Github. e.g. _cloudant.local.json_ and _couch.local.json_...